docs/java-rest/high-level/document/delete.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[java-rest-high-document-delete]]
+=== Delete API
+
+[[java-rest-high-document-delete-request]]
+==== Delete Request
+
+The most simple Delete Request needs is:
+
+["source","java",subs="attributes,callouts"]
+--------------------------------------------------
+sys2::[perl -ne 'exit if /end::delete-request/; print if $tag; $tag = $tag || /tag::delete-request/' {docdir}/../../client/rest-high-level/src/test/java/org/elasticsearch/client/DocumentationIT.java]
```

Comment:
I wonder if it is worth dumping the script into the root of the doc directory so you don't have to repeat it? On second thought, maybe just add it like it is now and I'll see about making a followup? I have some ideas - maybe write an asciidoc macro or something.